hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationRegistry.java

```diff
@@ -107,7 +108,7 @@ public boolean register(Invocation invocation) {
         final long callId;
         try {
             boolean force = invocation.op.isUrgent() || invocation.isRetryCandidate();
-            callId = callIdSequence.next(force);
+            callId = force ? callIdSequence.forceNext() : callIdSequence.next();
         } catch (TimeoutException e) {
```

Comment:
Same exception caught and then converted.. Makes sense to just throw the right exception if possible. Perhaps the exception message is a pita.